spring-transaction/src/test/kotlin/org/jetbrains/exposed/spring/SpringTransactionEntityTest.kt

```diff
@@ -59,36 +68,54 @@ open class Service {
     open fun findOrderByProduct(product: String): OrderDAO? {
         return OrderDAO.find { OrderTable.product eq product }.singleOrNull()
     }
+
+    open fun transaction(block: () -> Unit) {
+        block()
+    }
+
+    open fun cleanUp() {
+        SchemaUtils.drop(CustomerTable, OrderTable)
+    }
 }
 
 open class SpringTransactionEntityTest : SpringTransactionTestBase() {
 
     @Autowired
     lateinit var service: Service
 
-    @Test @Commit
-    open fun test01() {
-        transaction {
-            SchemaUtils.create(CustomerTable, OrderTable)
-        }
+    @Autowired
+    lateinit var ds: EmbeddedDatabase
```

Comment:
This seems to be unused. If so, could you please remove it and any unused imports?